docs/messages.md

```diff
@@ -0,0 +1,25 @@
+# Messages
+
+Messages, located in the Messages Inbox, are used to notify users about important information
+
+Technically, they are implemented as add-ons (see the add-on documentation).
+
+The JSON files are validated via [JSON Schema](https://json-schema.org/) files:
+https://github.com/mozilla-mobile/mozilla-vpn-client/tree/main/scripts/ci/jsonSchemas
+
+On this page, we want to describe how to write a message.
+
+## JSON format
+
+The Message JSON files are add-on JSON objects, with type "message" and a "message"
+property object with the following properties:
+
+| Property | Description  | Type | Required |
+| -- | -- | --| -- |
+| id | A unique identifier for the message | String | Yes |
+| title | The subject of the message  | String | Yes |
+| subtitle | A brief description of the message | String | No |
+| title_comment | An optional comment to describe the meaning of the title | String | No |
```

Comment:
Is `title_comment` used exclusively by l10n?

Reply:
Yes and we should have subtitle_comment too